lib/matplotlib/backends/wx_compat.py

```diff
@@ -145,12 +145,23 @@ def _AddTool(parent, wx_ids, text, bmp, tooltip_text):
             kind = wx.ITEM_CHECK
         else:
             kind = wx.ITEM_NORMAL
-        parent.AddTool(wx_ids[text], label=text,
-                       bitmap=bmp,
-                       bmpDisabled=wx.NullBitmap,
-                       shortHelpString=text,
-                       longHelpString=tooltip_text,
-                       kind=kind)
+
+        if LooseVersion(wx.VERSION_STRING) >= LooseVersion("4.0.0b2"):
```

Comment:
Would all of these work for the old wx as well?  In other words, could we just drop the code in the else clause if we decide not to support the beta versions of phoenix (once the actual release of phoenix comes)?

Reply:
Yes. At that point I think there could also be a bit more simplification for Classic wxPython support as well, since the new `AddTool` signature matches the signature for `DoAddTool` in Classic. So it would then boil down to just a difference in method names.

Reply:
Can you change the PR accordingly?